docs/source/onedal/gpu_support.rst

```diff
@@ -0,0 +1,59 @@
+.. ******************************************************************************
+.. * Copyright 2023 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _onedal_gpu_support:
+
+CPU and GPU Support
+===================
+Not all computation modes, methods, and parameters are supported on both CPU and GPU.
+Differences in CPU and GPU support are listed below.
+
+GPU supported targets
+*********************
+oneDAL library targets Intel GPUs support only, while technically it should be feasible
+to run on other HW platforms assuming SYCL runtime availability for platform.
+
+.. tabularcolumns::  |\Y{0.5}|\Y{0.5}|
+
+.. list-table:: GPU support conciderations
+   :header-rows: 1
+   :widths: 10 10
+   :align: left
+
+   * - Algorithm
+     - GPU support
+   * - :ref:`alg_kmeans`
+     - 
+   * - :ref:`alg_covariance`
+     - 
+   * - :ref:`alg_pca`
+     - 
+   * - :ref: Logistic Regression
+     - 
+   * - :ref: Linear Regression
+     - 
+   * - :ref:`alg_df`
+     - 
+   * - :ref: Extra Trees 
```

Comment:
Are these intended to be marked with `` (for Logistic, Linear, Extra Trees)?

Reply:
there should be documentation placed for those so we can link to them.

Reply:
What do these look like currently in the docs once they are built?

Reply:
<img width="466" alt="image" src="https://github.com/oneapi-src/oneDAL/assets/18015221/3b143d87-ee15-498a-80e1-7614dd5d6fa6">

But with ref word cleaned up - so no link